feature/aft/aft_base/otg_tests/afts_base/README.md

```diff
@@ -42,7 +36,7 @@ Establish RSVP Sessions between ATE:port3 and SUT:port3.
 
 *   BGP route advertised from ATE:port1,port2 must have 2 nexthops.
 *   IS-IS route advertised from ATE:port3 must have one next hop.
-*   BGP route advertised from ATE:port3 must have 2 next hops pointing to tunnels.
+*   BGP route advertised from ATE:port3 must have indirect next hop.
 *   Use gnmi Subscribe with ON_CHANGE option to /network-instances/network-instance/afts.
 *   For verifying prefix, nexthop groups, next hop use the leaves mentioed in the path section.
 *   Verify afts prefix advertised by BGP,ISIS.
```

Comment:
Please expand on this "verify" -- does this mean we check that the AFT prefixes are exactly what was injected? 

Especially for "verify all other leaves mentioned in the path section", this seems very ambiguous. Please consider writing this README as though you are trying to plan what code you're going to write. We don't need to enumerate exactly the check in prose, but we do need to define the correctness criteria.